src/main/java/com/internet/shop/controllers/RegistrationController.java

```diff
@@ -32,6 +34,11 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp)
         String passwordRepeat = req.getParameter("pwd-repeat");
         if (password.equals(passwordRepeat)) {
             User user = new User(name, login, password);
+            if (user.getLogin().contains("admin")) {
+                user.setRoles(Set.of(Role.of("ADMIN")));
+            } else {
+                user.setRoles(Set.of(Role.of("USER")));
+            }
```

Comment:
first, admin cannot have a shopping cart
also, the user can choose a login that contains `admin`

let's create the admin user in InjectDataController instead
this controller may be accessed with a button from the main menu, for example